app/models/custom_field.rb

```diff
@@ -1,19 +1,18 @@
+# frozen_string_literal: true
+
 class CustomField < ApplicationRecord
   auto_strip_attributes :name, nullify: false
   validates :name,
             presence: true,
             length: { maximum: Constants::NAME_MAX_LENGTH },
-            uniqueness: { scope: :team, case_sensitive: true },
+            uniqueness: { scope: :team_id, case_sensitive: true },
```

Comment:
Shoulda matchers has bug: https://github.com/thoughtbot/shoulda-matchers/issues/814 Without `_id` below test syntax for validate uniquness not works. 
`expect(custom_field).to validate_uniqueness_of(:name).scoped_to(:team_id)`

I am not sure whick one is better/right, but we have to test in a different way (if we want test unqinuess to scope validation) to avoid `_id`s. 

@radiokills said that `_id` is ok or even better.